README.md

```diff
@@ -20,26 +20,31 @@ Or install it yourself as:
 
 ## Usage
 
-### Sinatra instructions
-#### App setup
-* Add to Gemfile as well as listen 3.1.1
-* Require in server file (i.e. config.ru)
```

Comment:
I'm thinking about how I would do that. I'd target sinatra and rails at first, and welcome any contributions.  In both cases, I'll want inquire the app about what it's Opal paths are and then start the reloader w/those directories.  For Rails, I'll want to only automatically start it client/server side if Rails.env.development?